libcontainer/integration/utils_test.go

```diff
@@ -19,6 +20,36 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var busyboxTar string
+
+// init makes sure the container images are downloaded,
+// and initializes busyboxTar. If images can't be downloaded,
+// we are unable to run any tests, so panic.
+func init() {
```

Comment:
This function looks a tad complicated, but all it does is calls `get-images.sh`
and parses its output to get the value of `BUSYBOX_IMAGE`. This way we
don't have to worry about doing some extra steps before running `go test`.